linebot/api.py

```diff
@@ -1034,6 +1034,88 @@ def get_insight_message_event(self, request_id, timeout=None):
 
         return InsightMessageEventResponse.new_from_json_dict(response.json)
 
+    def issue_channel_access_token_v2_1(
+            self, client_assertion, grant_type='client_credentials',
+            client_assertion_type='urn:ietf:params:oauth:client-assertion-type:jwt-bearer',
+            timeout=None):
+        """Issues a channel access token v2.1.
+
+        https://developers.line.biz/en/reference/messaging-api/#issue-channel-access-token-v2-1
+
+        :param str client_assertion: Clint assertion.
+        :param str grant_type: `client_credentials`
+        :param str client_assertion_type: `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`.
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.IssueChannelTokenResponse`
+        :return: IssueChannelTokenResponse instance
```

Comment:
[imo]
I think that the return description is unnecessary for consistency with other descriptions. (`rtype` alone is enough)